src/pages/home/components/HotPrices/HotPrices.scss

```diff
@@ -0,0 +1,62 @@
+.hotPrices {
```

Comment:
It should be a universal component, so name the file properly (CardsSlider or etc.)